### PR TITLE
Add ci

### PR DESCRIPTION
**Pull request by tomkoren21**

### CODE REVIEW

**Comment by tomkoren21** [comment]
I really think we need CI here - discussed this with Idan. This is a relatively simple setup, but it works. LMK what you think

**Review by tip3x — APPROVED** [review]
LGTM! Finally!

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+          installer-parallel: true
+      - name: Load cached venv
+        id: cached-poetry-dependencies
+        uses: actions/cache@v2
+        with:
+          path: .venv
+          key: venv-${{ runner.os }}-${{ steps.setup-python.outputs.python-version }}-
+      - name: Install dependencies
+        if: steps.cached-poetry-dependencies.outputs.cache-hit != 'true'
+        run: poetry install --no-interaction --no-root
```
Comment (doronHarnoy) [inline .github/workflows/ci.yml:33]: cool! so poetry's venv will be written to cache and not be created again every time?

File: test/models/test_mnasnet.py [hunk 2]
```diff
@@ -1,28 +1,16 @@
 import numpy as np
 import pytest
 from test.utils import convert_and_test
-from torchvision.models import mnasnet0_5, mnasnet1_0, mnasnet0_75, mnasnet1_3
+from torchvision.models import mnasnet0_5, mnasnet0_75
```
Comment (doronHarnoy) [inline test/models/test_mnasnet.py:4]: why did we keep here both mnasnet0_5 and mnasnet0_75 and removed the other two?

**Review by doronHarnoy — APPROVED** [review]
Great job!

File: test/models/test_mnasnet.py (hunk 2 above)
Comment (tomkoren21) [reply test/models/test_mnasnet.py:4]: I kept only the 0_5 in a later commit, but forgot to remove the import. will do now.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (tomkoren21) [reply .github/workflows/ci.yml:33]: I think so, but to be honest didn't deep-dive into it, rather copied the gist from some example. I suggest we optimize and test the caching when it becomes an issue

File: test/models/test_mnasnet.py (hunk 2 above)
Comment (doronHarnoy) [reply test/models/test_mnasnet.py]: I agree, i ask because i think it can improve other ci tests that we have that dont use that mechanism

**Pull request merged**